.github/workflows/create-or-update-release-pr.yml

```diff
@@ -49,7 +49,7 @@ jobs:
           git config user.email 197425009+otelbot@users.noreply.github.com
           npm run github:create_or_update_release_pr
         env:
-          GH_TOKEN: ${{ steps.otelbot-token.outputs.token }}
+          GITHUB_TOKEN: ${{ steps.otelbot-token.outputs.token }}
```

Comment:
I think this was a copy-paste error. We expect `GITHUB_TOKEN` here (that's what the gh cli uses).